src/Meplato.Store2.Tests/Availabilities/DeleteTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿#region Copyright and terms of services
+
+// Copyright (c) 2015 Meplato GmbH.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License. You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License
+// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+// or implied. See the License for the specific language governing permissions and limitations under
+// the License.
+
+#endregion
+
+using System.Threading.Tasks;
+using NUnit.Framework;
+
+namespace Meplato.Store2.Tests.Products
```

Comment:
Build error, wrong namespace

```suggestion
namespace Meplato.Store2.Tests.Availabilities
```